awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsSpanProcessingUtil.java

```diff
@@ -40,6 +47,19 @@ final class AwsSpanProcessingUtil {
   static final String LOCAL_ROOT = "LOCAL_ROOT";
   static final String SQS_RECEIVE_MESSAGE_SPAN_NAME = "Sqs.ReceiveMessage";
   static final String AWS_SDK_INSTRUMENTATION_SCOPE_PREFIX = "io.opentelemetry.aws-sdk-";
+  // Max keyword length supported by parsing into remote_operation from DB_STATEMENT
+  static final int MAX_KEYWORD_LENGTH = 27;
+  private static final String SQL_DIALECT_KEYWORDS_JSON = "configuration/sql_dialect_keywords.json";
+
+  static List<String> getDialectKeywords() throws IOException {
+    InputStream jsonFile =
+        AwsSpanProcessingUtil.class.getClassLoader().getResourceAsStream(SQL_DIALECT_KEYWORDS_JSON);
+    ObjectMapper mapper = new ObjectMapper();
+    JsonNode jsonNode = mapper.readValue(jsonFile, JsonNode.class);
+    JsonNode arrayNode = jsonNode.get("keywords");
+    ObjectReader reader = mapper.readerFor(new TypeReference<List<String>>() {});
+    return reader.readValue(arrayNode);
+  }
```

Comment:
Related to my comment about using the json file and reading it, this logic could be much simpler if we are using a native datatype (`Set` maybe?) to store the db statement keywords.

Reply:
Oh. just realized that the keywords need to be ordered, so can't use `Set`. A simple `List` then, although it will not deduplicate the elements.

Reply:
The keywords in the list are supposed to be deduplicated already.